README.md

```diff
@@ -2,10 +2,10 @@
 Concordia Campus Guide application - SOEN390
 
 
-## Objective:
+## Objective
```

Comment:
I'll let it slide this time but in the future put these kinds of changes in their own pull request :)

Reply:
Got it! I decided to do have it in this PR since they're just minor styling changes and I didn't feel like they deserved a code review for it, but I'll keep that in mind!